app/lib/features/pins/widgets/pin_item.dart

```diff
@@ -87,7 +88,12 @@ class _PinItemState extends ConsumerState<PinItem> {
               formkey: _formkey,
             ),
             const SizedBox(height: 20),
-            _buildAttachmentList(),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: <Widget>[
+                _buildAttachmentList(),
+              ],
+            ),
```

Comment:
Can I have more details about this change?

Reply:
Oh it shouldn't be here, was added during one refactoring approach. Let me remove it!